src/styles/components.css

```
@@ -0,0 +1,478 @@
+
+.privacy-page {
+  @apply flex min-h-screen items-start justify-center bg-gradient-to-b from-lightbg to-lightSidebar px-6 py-16 dark:from-darkbg dark:to-darkSidebar;
+}
+
+.privacy-card {
+  @apply w-full max-w-3xl rounded-3xl border border-white/30 bg-white/60 p-10 shadow-2xl backdrop-blur-xl transition-all duration-300 dark:border-white/10 dark:bg-black/40 sm:p-14;
+}
+
+.privacy-title {
+  @apply mb-4 text-center text-5xl font-extrabold tracking-tight;
+  background-image: linear-gradient(to right, #0cacf0, #3b82f6);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)